docs/usage/shoot_high_availability.md

```diff
@@ -0,0 +1,52 @@
+# Highly Available Shoot Control Plane
```

Comment:
As discussed, please link this document in https://github.com/gardener/gardener/blob/master/docs/README.md.